include/lbann/layers/regularizers/local_response_normalization.hpp

```diff
@@ -207,19 +207,18 @@ class local_response_normalization_layer : public regularizer_layer<TensorDataTy
     if (local_input.Height() > 0 && local_input.Width() > 0) {
       const ScalingType zero = El::TypeTraits<ScalingType>::Zero();
       const ScalingType one = El::TypeTraits<ScalingType>::One();
-      CHECK_CUDNN(cudnnLRNCrossChannelBackward(cudnn::get_handle(),
-                                               m_lrn_cudnn_desc,
-                                               CUDNN_LRN_CROSS_CHANNEL_DIM1,
-                                               &one,
-                                               m_tensors_cudnn_desc.get_prev_activations(),
-                                               local_input.LockedBuffer(),
-                                               m_tensors_cudnn_desc.get_prev_error_signals(),
-                                               local_gradient_wrt_output.LockedBuffer(),
-                                               m_tensors_cudnn_desc.get_activations(),
-                                               local_output.LockedBuffer(),
-                                               &zero,
-                                               m_tensors_cudnn_desc.get_error_signals(),
-                                               local_gradient_wrt_input.Buffer()));
+      cudnn::lrn_cross_channel_backward(
+        m_lrn_cudnn_desc,
+        one,
+        m_tensors_cudnn_desc.get_prev_activations(),
+        local_input,
+        m_tensors_cudnn_desc.get_prev_error_signals(),
+        local_gradient_wrt_output,
+        m_tensors_cudnn_desc.get_activations(),
+        local_output,
```

Comment:
Looks like we had a bug before. It should be:
```suggestion
        one,
        m_tensors_cudnn_desc.get_activations(),
        local_output,
        m_tensors_cudnn_desc.get_prev_error_signals(),
        local_gradient_wrt_output,
        m_tensors_cudnn_desc.get_prev_activations(),
        local_input,
```
I find it interesting that AlexNet was working before with this error... I suppose there's a reason LRN went out of fashion.